test/system/070-build.bats

```diff
@@ -887,4 +887,61 @@ function teardown() {
     basic_teardown
 }
 
+@test "podman build .dockerignore failure test " {
+    tmpdir=$PODMAN_TMPDIR/build-test
+    subdir=$tmpdir/subdir
+    mkdir -p $subdir
+
+    cat >$tmpdir/.dockerignore <<EOF
+*
+subdir
+!*/sub1*
+EOF
+    cat >$tmpdir/Containerfile <<EOF
+FROM $IMAGE
+COPY ./ ./
+COPY subdir ./
+EOF
+    run_podman 125 build -t build_test $tmpdir
+    is "$output" ".*Error: error building at STEP \"COPY subdir ./\": no items matching glob" "Expected to fail"
+}
+
+@test "podman build .containerignore and .dockerignore test " {
+    tmpdir=$PODMAN_TMPDIR/build-test
+    mkdir -p $subdir
+    touch $subdir/test1 $subdir/test2
+    cat >$tmpdir/.containerignore <<EOF
+test2*
+EOF
+    cat >$tmpdir/.dockerignore <<EOF
+test2*
+EOF
+    cat >$tmpdir/Containerfile <<EOF
+FROM $IMAGE
+COPY ./ ./test/
+RUN ls /tmp/test/
+EOF
+    run_podman build -t build_test $tmpdir
+    is "$output" ".*test2" "test2 should exists in the DockerfileExpected to fail"
+}
+
+@test "podman build .containerignore and .dockerignore test " {
```

Comment:
Could you give this test a unique name? (It's the same as line 909)

Reply:
Removed since it was a duplicate of prior.